public/main.js

```diff
@@ -49,18 +48,24 @@ function getTodoList(callback) {
     createRequest.send();
 }
 
-function makeButton(content, cssClass, onClick) {
+function makeButton(content, onClick) {
     var button = document.createElement("button");
     button.textContent = content.toString();
-    button.className = cssClass.toString();
+    button.className = "button";
     button.addEventListener("click", onClick);
     return button;
 }
 
 function updateIncompletes(todos, countLabel) {
+    countLabel.textContent = "";
+    var anyComplete = false;
     countLabel.textContent = todos.filter(function(todo) {
+        if (todo.isComplete) {anyComplete = true;}
```

Comment:
I feel like this would read better if you first counted the incomplete todos with reduce and then calculated if there were any complete todos by comparing the number of todos with the number of incomplete todos.